src/Actions/UpdateUserProfileInformation.php

```diff
@@ -23,7 +24,7 @@ class UpdateUserProfileInformation implements UpdatesUserProfileInformation
     public function update($user, array $input)
     {
         Validator::make($input, [
-            'name'  => ['required', 'string', 'max:255', resolve(PoliteUsername::class)],
+            'name'  => ['required', 'string', 'min:3', 'max:30', resolve(PoliteUsername::class), resolve(AllowedCharactersUsername::class)],
```

Comment:
this is for `name`, not `username` so it shouldn't care about that. on MSQ this is not used as we have a custom one for username/email combination instead